storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/RemoteLogMetadataCache.java

```diff
@@ -115,6 +115,9 @@ public boolean isInitialized() {
         return initializedLatch.getCount() == 0;
     }
 
+    public RemoteLogMetadataCache() {
```

Comment:
Is this necessary? the constructor without parameters is the default constructor to this class.